firstvoices/backend/serializers/membership_serializers.py

```diff
@@ -1,3 +1,4 @@
+from jwt_auth.models import User
```

Comment:
This is a little Django gotcha. Instead of directly importing the User model, please use the get_user_model() method

https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#referencing-the-user-model